File: app/src/main/java/vinova/henry/com/hotfilm/feature/detail/IDetailContract.java
package vinova.henry.com.hotfilm.feature.detail;


import android.view.View;

import java.util.List;

import vinova.henry.com.hotfilm.models.Trailer;

public interface IDetailContract {
    interface IDetailView{
        void onClickItem(int position);

    }
    interface IDetailPresenter{
        String[] createSpinnerItem(List<Trailer> trailers);
    }
}

File: app/src/main/java/vinova/henry/com/hotfilm/feature/detail/IDetailContract.kt
package vinova.henry.com.hotfilm.feature.detail


import vinova.henry.com.hotfilm.models.Trailer

interface IDetailContract {
    interface IDetailView {
        fun onClickItem(position: Int)

    }

    interface IDetailPresenter {
        fun createSpinnerItem(trailers: List<Trailer>): Array<String>
    }
}
